app/assets/stylesheets/application.scss

```
@@ -29,18 +29,45 @@
   height:40px;
   object-fit:cover;
   border-radius:50%;
-  margin-top: 8px;
-  margin-bottom: 7px;
-  margin-right: 10px;
-  margin-left: 10px;
+  margin: 8px 10px 7px 10px
 }
 .header_notification_icon{
   font-size: 45px;
   margin-top: 5px;
 }
 .header_login_button{
-  margin-right: 10px;
-  margin-left: 10px;
-  margin-top: 5px;
-  margin-bottom: 5px;
+  margin: 5px 10px 5px 10px
```

下記のほうがシンプルに書けて良いと思います（他の箇所も必要に応じて修正を）
```suggestion
  margin: 5px 10px
```